frontend/src/components/CreateEmployee/CreateEmployee.tsx

```diff
@@ -44,6 +50,8 @@ const CreateEmployee: React.FC = () => {
       country: '',
       title: '',
       hourlyWage: '',
+      businessId: '',
+      isAdmin: account.admin ? account.admin : false,
```

Comment:
Wouldn't it be better here to just put "isAdmin: account.admin"? Since if it's true then it will be true and if it's false it will be false.

Reply:
The reason I did this is because the return value from the backend can be an empty string. I want `isAdmin` to be an actual Boolean in the frontend.